### ISSUE TITLE
Browser & Engine Sniffing

### ISSUE THREAD

**Issue by MrSaints** [TurbineCSS/Turbine#144]
On a Chrome,

Turbine CSS-P:

```
#foo
    browser:firefox
    color:green
#foo
    browser:chrome
    color:red
```

Would show a red text for element with ID 'foo'.
On a Firefox, it does not show the colour green or any styles.
However, if these two element styles are flipped around to give:

```
#foo
    browser:chrome
    color:red
#foo
    browser:firefox
    color:green
```

It would display a green colour text for element with ID 'foo' perfectly.

**Comment by SirPepe** [comment]
This happens because in version 1.x the two `#foo` rules are merged before the browser sniffer becomes active. This will be fixed in version 1.1.

**Comment by MrSaints** [comment]
Yea, I thought so. Great work on Turbine by the way. I'll be anticipating the 1.1 release.
EDIT: Sorry, accidentally closed the issue. I presume you are already aware of this anyway.
